### PR TITLE
ENH: numerically stable rolling_skew and rolling_kurt

### LINKED ISSUE #6929: PERF: Poor numerical stability of rolling_kurt and rolling_skew
The numerical stability of the current implementation of `rolling_kurt` and `rolling_skew` is very poor:

```
In [14]: x = np.random.rand(10)

In [15]: pd.Series(x).rolling(4).skew()
Out[15]:
0         NaN
1         NaN
2         NaN
3   -0.971467
4    0.596391
5   -1.163135
6   -0.611469
7   -1.624713
8   -0.021832
9    0.754653
dtype: float64

In [16]: pd.Series(x + 5000).rolling(4).skew()
Out[16]:
0         NaN
1         NaN
2         NaN
3   -0.968964
4    0.608522
5   -1.171751
6   -0.619354
7   -1.628942
8   -0.033949
9    0.741304
dtype: float64

In [17]: pd.Series(x).rolling(4).kurt()
Out[17]:
0         NaN
1         NaN
2         NaN
3    2.030185
4   -2.647975
5    1.183004
6   -2.178587
7    2.658058
8   -4.258505
9    0.939264
dtype: float64

In [18]: pd.Series(x + 100).rolling(4).kurt()
Out[18]:
0         NaN
1         NaN
2         NaN
3    2.030230
4   -2.648801
5    1.182382
6   -2.178574
7    2.657934
8   -4.258813
9    0.938661
dtype: float64
```

This can be solved using an updating algorithm, similar to what is done in #6817 for `rolling_var`.

### PR DESCRIPTION
**Pull request by jaimefrio**
Close #6929. Hat tip to @behzadnouri for the z-scores idea in #8270, and to @seth-p for the general structure of the code, separating removal and addition of observations.

### CODE REVIEW

**Comment by jreback** [comment]
@jaimefrio can you rebase/revisit ?

**Comment by jreback** [comment]
@jaimefrio can you rebase and add a release note, lgtm otherwise.

**Comment by jreback** [comment]
@jaimefrio can you rebase / revist ?

**Comment by jaimefrio** [comment]
Yes, will do, @jreback. Sorry I missed your earlier pings!

**Comment by jreback** [comment]
@jaimefrio can you update for 0.17.0

**Comment by jreback** [comment]
can you update?

**Comment by jreback** [comment]
@jaimefrio can you rebase?

**Comment by jreback** [comment]
@jaimefrio can you rebase

**Comment by jaimefrio** [comment]
I've rebased, but I don't remember anymore what was missing for this to be ready...

File: pandas/stats/moments.py [hunk 1]
```diff
@@ -374,6 +375,11 @@ def _rolling_moment(arg, window, func, minp, axis=0, freq=None, center=False,
         Passed on to func
     kwargs : dict
         Passed on to func
+    center_data : bool
+        If True, subtract the mean of the data from the values
+    norm_data: bool
```
Comment (jreback) [inline pandas/stats/moments.py]: can you add a versionadded directiver here

**Comment by jreback** [comment]
can you add the examples from top of the issue #6929 e.g. validating the problem skew/kurt results that we were getting before (IOW, calculate and compare to what they should be)

**Comment by jreback** [comment]
can you add a release note in v0.17.0 (maybe a separate mini-section), to show what has changed? (e.g. you can use the same examples that show the stability problem).

**Comment by jreback** [comment]
@jaimefrio can you update

**Comment by jreback** [comment]
can you add a release note to 0.17.1

**Comment by jreback** [comment]
@jaimefrio can you rebase/update?

**Comment by jreback** [comment]
closing, but pls reopen when  you can update

**Pull request closed without merging**